tokio/Cargo.toml

```diff
@@ -36,6 +36,7 @@ default = [
 #  "timer",
 #  "udp",
 #  "uds",
+  "macros",
```

Comment:
I would not add a feature for macros. It already is included as part of `rt-full`. We can refine those details later.

Reply:
Done